packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -2271,14 +2265,15 @@ export let tableTests = () => {
         fireEvent.keyDown(getCell(tree, 'Baz 10'), {key: 'End', shiftKey: true, ctrlKey: true});
 
         let expected = [];
-        for (let i = 10; i <= 100; i++) {
+        for (let i = 10; i <= 30; i++) {
           expected.push('Foo ' + i);
         }
 
         checkSelection(onSelectionChange, expected);
       });
 
-      it('should extend a selection with Shift + PageDown', function () {
+      // Skip the pagedown/up ones when we aren't mocking client height since those rely on math to calculate where to move focus
+      it.skip('should extend a selection with Shift + PageDown', function () {
```

Comment:
Is the path forward that these tests need height defined or we tell people they have to mock a height to do these, since you're saying when there is no height value to use for calculations these won't work.

Reply:
IMO these kinds of tests would need a mock height defined by the user. I don't know how often people will be testing this keyboard navigation though tbh, ideally they would just go ahead and find the row they want to interact with since they should be able to trust that we are testing the interactions on our side